phac-nml/irida-next
# Dev: Enhance Seeding Sample Attachments

## What does this PR do and why?
_Describe in detail what your merge request does and why._

Updated seeding sample attachments to only store each unique file as a single blob, instead of a blob per file per sample. This should result in quicker seeding and using less storage on machine.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Run `./scripts/clean_dev.sh`
2. Launch server and navigate to a sample
3. Observe that 2 attachments are present (pair end data)

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
